_includes/manuals/1.10/4.9.1_global_status.md

```diff
@@ -0,0 +1,17 @@
+The global status represents the overall status of a particular host. The
+status can have one of three possible values - **OK**, **Warning**, **Error**.
+OK means that all sub-statuses do not report anything wrong. It is represented
```

Comment:
This section could do with being in separate paragraphs or bullet points for each status.

"OK means that no errors were reported by any sub-status"?